src/Services/FHIR/FhirValueSetService.php

```diff
@@ -0,0 +1,184 @@
+     *  | abook_type                      | spe               | Specialist            |  20 |          0 |            2 |         | NULL  |       |
+     *  | abook_type                      | vendor            | Vendor                |  25 |          0 |            3 |         | NULL  |       |
+     *  | address-types                   | both              | Postal & Physical     |  30 |          0 |            0 |         | NULL  |       |
+     *  | address-types                   | physical          | Physical              |  20 |          0 |            0 |         | NULL  |       |
+     *  | address-types                   | postal            | Postal                |  10 |          0 |            0 |         | NULL  |       |
+     *  | address-uses                    | billing           | Billing               |  50 |          0 |            0 |         | NULL  |       |
+     *  | address-uses                    | home              | Home                  |  10 |          0 |            0 |         | NULL  |       |
+     *  | address-uses                    | old               | Old/Incorrect         |  40 |          0 |            0 |         | NULL  |       |
+     *  | address-uses                    | temp              | Temporary             |  30 |          0 |            0 |         | NULL  |       |
+     *  | address-uses                    | work              | Work                  |  20 |          0 |            0 |         | NULL  |       |
+     *  | adjreason                       | Adm adjust        | Adm adjust            |   5 |          0 |            1 |         | NULL  |       |
+     *  | adjreason                       | After hrs calls   | After hrs calls       |  10 |          0 |            1 |         | NULL  |       |
+     *  | adjreason                       | Bad check         | Bad check             |  15 |          0 |            1 |         | NULL  |       |
+     *  | adjreason                       | Bad debt          | Bad debt              |  20 |          0 |            1 |         | NULL  |       |
+     *  | adjreason                       | Coll w/o          | Coll w/o              |  25 |          0 |            1 |         | NULL  |       |
+     *
+     */
+
+
+
+    const USCGI_PROFILE_URI = 'http://hl7.org/fhir/StructureDefinition/shareablevalueset';
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->appointmentService = new AppointmentService();
+        $this->listOptionService = new ListService();
+    }
+
+    /**
+     * Returns an array mapping FHIR Resource search parameters to OpenEMR search parameters
+     */
+    protected function loadSearchParameters()
+    {
+        return [
+        '_id' => new FhirSearchParameterDefinition('_id', SearchFieldType::TOKEN, [new ServiceField('uuid', ServiceField::TYPE_UUID)]),
```

Comment:
Is this actually working for you when you do a call to a specific value set?  All of your id values that are returned appear to be string values and are not uuids from what I see in the code.  

Seems like you will need to use this line here:
```
'_id' => new FhirSearchParameterDefinition('_id', SearchFieldType::TOKEN, [new ServiceField('id', ServiceField::TYPE_STRING)]),

Reply:
I can see that you aren't filtering on this value, so this is likely why it didn't blow up on you earlier.  Still needs to be fixed.

Reply:
yep, just forgot to go back and do that.